src/OpenTelemetry.Instrumentation.AdditionalDeps/OpenTelemetry.Instrumentation.AdditionalDeps.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.1;net5.0</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="6.0.0" />
+  </ItemGroup>
+
+  <Target Name="CallComposeStore" AfterTargets="Publish">
+    <PropertyGroup>
+      <AdditionalDepsPath>$(OutputDir)\AdditionalDeps</AdditionalDepsPath>
+      <StoreOutputBasePath>$(OutputDir)\store\</StoreOutputBasePath>
+      <AdditionalDepsPath Condition="'$(TracerHomePath)' != ''">$(TracerHomePath)\AdditionalDeps</AdditionalDepsPath>
+      <StoreOutputBasePath Condition="'$(TracerHomePath)' != ''">$(TracerHomePath)\store</StoreOutputBasePath>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <StoreRuntime Include="x86">
+        <PlatformTargetForStore>x86</PlatformTargetForStore>
+        <StorePath>$(StoreOutputBasePath)</StorePath>
+      </StoreRuntime>
+      <StoreRuntime Include="x64">
+        <PlatformTargetForStore>x64</PlatformTargetForStore>
+        <StorePath>$(StoreOutputBasePath)</StorePath>
+      </StoreRuntime>
+    </ItemGroup>
+    
+    <Message Text="Calling ComposeStore for TargetFramework='$(TargetFramework)'" Importance="high" />
+    
+    <ItemGroup>
+      <ProjectsToBuild Include="Package.xml">
+        <Properties>TargetFramework=$(TargetFramework);PlatformTarget=%(StoreRuntime.PlatformTargetForStore);RuntimeIdentifier=%(StoreRuntime.Identity);SkipOptimization=true;ComposeDir=%(StoreRuntime.StorePath);CreateProfilingSymbols=false;Version=$(SemanticVersion4parts);NuGetPackageRootOriginal=$(NuGetPackageRoot);Environment=$(Environment)</Properties>
```

Comment:
These are understandable by their names but is there any doc with the full set options? It looks like it is a call to `dotnet store` but some of the names don't match exactly with those. If there is a doc with more info it seems worth a link here.

Reply:
Related Q.: `RuntimeIdentifier` feels like a `RID` but here is just the platform target? Will this work if there is a dependency to a specific RID?

Reply:
You're correct. It is a call to dotnet store, did not find a doc on how to call from the project file. I will leave a comment stating it is a call to dotnet store.

Yes, RID here is a platform target. It won't work if there is a dependency to specific RID. If we want to target a specific target then the runtime store folders needs to be created for those many RIDs that are referenced.